tests/e2e/etcd_config_test.go

```diff
@@ -308,5 +308,7 @@ func TestGrpcproxyAndCommonName(t *testing.T) {
 	if err != nil {
 		t.Errorf("Unexpected error: %s", err)
 	}
-	p.Stop()
```

Comment:
Could you please write what's the output now when the /bin/etcd file does not exists ? 
It should clearly fail and not attempt to continue. 

I would consider: 
```
  defer func() {
     if p!=nil { p.Stop() }
  }
  if err != nil {
     t.Fatalf()
  }
```

Reply:
ok, thanks.

Reply:
defer needs to be called before Fatal(...)